runtime/binding-mqtt/src/main/java/io/aklivity/zilla/runtime/binding/mqtt/internal/stream/MqttServerFactory.java

```diff
@@ -2090,7 +2096,7 @@ private int onDecodeConnect(
             DirectBuffer buffer,
             int progress,
             int limit,
-            MqttConnectHeaderFW connectHeader,
+            MqttConnectFW connect,
```

Comment:
Pass the parts of `connect` as individual parameters so this can be called easily from `decodeConnectV4` and `decodeConnectV5`.